### ISSUE TITLE
⚠ WIKI STANDARDS ⚠

### ISSUE THREAD

**Issue by Kitzunu** [azerothcore/wiki#528]
# WIKI STANDARDS

## FILE NAMING:

DATABASE TABLES files are to be named exactly as it is in the database,

ALL OTHER FILES are to be named in `lower case` and to use `-` (dashes) for spaces.

FILES SHOLD NOT contain any special characters other than `-` (dashes). If we want a file named C++ we name it cpp.

ALL WIKI FILES should end with with `.md`.

## DATABASE TABLE FILES:

ALL DATABASE TABLE FILES should be present in it's correct DATABASE FILE.

When adding/removing a table it should also be updated in `database-auth.md` `database-characters.md` `database-world.md`

## ARCHIVED PAGES:

ALL ARCHIVED PAGES that could potentially have some sort of value should be added in the `archive.md` list and links to these pages should be removed from other places in the wiki.

**Comment by pangolp** [comment]
Can't this issue be placed somewhere else, like in README.md or home.md, and close this issue?

**Comment by pangolp** [comment]
The information is fine, but I think it should be somewhere else, so that only the issues are available for suggestions or bug reports.

**Comment by Kitzunu** [comment]
No. It is done this way to have it pinned. Feel free to delete the comments to keep it clean.
